source/class/cv/ui/PageHandler.js

```diff
@@ -70,13 +70,17 @@ qx.Class.define('cv.ui.PageHandler', {
       }
       const currentPath = this.getCurrentPath();
       if (currentPath !== '') {
- qx.event.message.Bus.dispatchByName('path.'+currentPath+'.exitingPageChange', currentPath, target); 
-}
+        qx.event.message.Bus.dispatchByName('path.'+currentPath+'.exitingPageChange', currentPath, target);
+      }
 
-      const pageWidget = cv.ui.structure.WidgetFactory.getInstanceById(target);
+      let pageWidget = cv.ui.structure.WidgetFactory.getInstanceById(target);
 
       if (!pageWidget || !pageWidget.getDomElement()) { // check if page does exist
-        return;
+        if (currentPath !== '') {
```

Comment:
Open e.g. the demo config with an anchor in the URL to a non existing page.
E.g.: `?config=demo#id_4_`

Trying the same with the last RC it even shows a red error + Sentry

Reply:
But this is not the correct place to fix the problem, because if we do it like this we will have a mismatch between the page in the browser history (#id_4_) and the actual page shown (#id_).

I will append a patch to this PR that solves this earlier, by checking if the initial page exists and using `id_` as initial page of not. Then the browser history is in sync with whats actually show.